doc/api/deprecations.md

```diff
@@ -999,6 +999,15 @@ Type: Documentation-only
 The option `produceCachedData` has been deprecated. Use
 [`script.createCachedData()`][] instead.
 
+<a id="DEP00XX"></a>
+### DEP00XX: process.binding()
+
+Type: Documentation-only
+
+The `process.binding()` API is intended for use strictly by Node.js internal
```

Comment:
Nit: remove `strictly`

Reply:
Why?

Reply:
It's unnecessary. It's also imprecise. "only" would be more precise.

Reply:
So maybe this?:

> `process.binding()` is intended for use by Node.js internal code only.